Description: Don't install compatibility symlinks
 These links are shipped inside the scrotwm package on Debian.
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2013-02-09
Index: spectrwm.git/linux/Makefile
===================================================================
--- spectrwm.git.orig/linux/Makefile	2014-04-21 20:42:45.380047485 +0200
+++ spectrwm.git/linux/Makefile	2014-04-21 20:42:56.108047894 +0200
@@ -47,7 +47,6 @@
 	install -m 755 -d $(DESTDIR)$(MANDIR)/ru/man1
 	install -m 755 -d $(DESTDIR)$(XSESSIONSDIR)
 	install -m 755 spectrwm                    $(DESTDIR)$(BINDIR)
-	ln -sf spectrwm                            $(DESTDIR)$(BINDIR)/scrotwm
 	install -m 644 libswmhack.so.$(LIBVERSION) $(DESTDIR)$(LIBDIR)
 	ln -sf libswmhack.so.$(LIBVERSION)         $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBMAJORVERSION)
 	ln -sf libswmhack.so.$(LIBVERSION)         $(DESTDIR)$(LIBDIR)/libswmhack.so
@@ -60,7 +59,6 @@
 
 uninstall:
 	rm -f $(DESTDIR)$(BINDIR)/spectrwm
-	rm -f $(DESTDIR)$(BINDIR)/scrotwm
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBVERSION)
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBMAJORVERSION)
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so
